app/services/unsplash_service.rb

```diff
@@ -0,0 +1,21 @@
+class UnsplashService
+  def get_background_url(keyword)
+    search_query = keyword.gsub(",", "+")
```

Comment:
Not sure if this is necessary because I tried searching 'Los Angeles' and replaced search_query on line 7 with keyword and it still passed. However, I kept it for now in case something does break.